lib/fizzy/execute.cpp

```diff
@@ -499,6 +499,25 @@ inline bool invoke_function(const FuncType& func_type, uint32_t func_idx, Instan
     return true;
 }
 
+[[maybe_unused]] bool is_resize_exception(std::exception_ptr exception)
```

Comment:
Nice! Perhaps you want to add a comment here explaining our use case.

Reply:
Added a comment.